package.json

```diff
@@ -75,6 +76,7 @@
     "date-fns": "2.28.0",
     "dompurify": "2.3.4",
     "epubjs": "0.3.90",
+    "escape-html": "1.0.3",
```

Comment:
This is such a small utility maybe it's worth including it locally in jellyfin-web to avoid a "leftpad" situation?

Reply:
Do you mean to place https://github.com/component/escape-html/blob/master/index.js somewhere in `libraries`?
If there are no (license) problems to us.
_This utility was not updated for a while (it is perfect :stuck_out_tongue: )._

I am also thinking of replacing `htmlescape` with `escapeHtml` as it called in the utility. What do you think?

_offtopic_:
Is this the "leftpad" you're talking about?
https://github.com/left-pad/left-pad/blob/master/index.js

Reply:
Ah good call if we are already pulling it in for other dependencies it probably doesn't make sense to include it directly. I do think `escapeHtml` is better since it better denotes an action is being done.

_offtopic:_
I was referencing this incident where the leftpad package was nuked by the maintainer. :smile:  https://www.theregister.com/2016/03/23/npm_left_pad_chaos/